server/endpoints/chat.js

```diff
@@ -1,3 +1,4 @@
+const expresss = require("express");
```

Comment:
Why was this re-imported?

`chatEndpoints(app) {` `app` is the passed-in express server reference to build these endpoints, but I don't see where other references to express are required in this change